packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -43,14 +44,11 @@ export class AzureClient {
      * @param props - Properties for initializing a new AzureClient instance
      */
     constructor(private readonly props: AzureClientProps) {
-        this.urlResolver = new AzureUrlResolver(
-            this.props.connection.tenantId,
-            this.props.connection.orderer,
-            this.props.connection.storage,
-        );
+        this.urlResolver = new AzureUrlResolver();
         // The local service implementation differs from the Azure Fluid Relay in blob
         // storage format. Azure Fluid Relay supports whole summary upload. Local currently does not.
-        const enableWholeSummaryUpload = this.props.connection.tenantId !== LOCAL_MODE_TENANT_ID;
```

Comment:
it's not directly related to your changes, but why this code is here? Why it's not hidden in the driver, either in the form of some props / policy that changes behavior of a single driver, auto-detecting (based on endpoint uri/properties) or having 2 drivers (possibly reusing 99% of the code through inheritance / mixin)?

If it helps - ODSP driver has somewhat similar support for various differences in environments / workflows / staging of new features through auto-detection, policy provided by service, policy provided by runtime, etc. But it does not leak from driver

Reply:
I believe this was added in to allow the AzureClient to support connecting with both Tinylicious (which does not use antler) and FRS (which does). It uses the same driver and document factory to connect with both services, and then depending on  which one, supplies the appropriate `enableWholeSummaryUpload` boolean to the r11s' document service factory

Reply:
But it feels like the driver should auto-detect that case somehow (maybe with the help of service providing some policy) and do right thing, such that client code does not need to know about these details.